hibernate-core/src/test/java/org/hibernate/test/dialect/functional/OracleDialectSequenceInformationTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.test.dialect.functional;
+
+import static org.hibernate.testing.transaction.TransactionUtil.doInAutoCommit;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
```

Comment:
Hibernate's code convention is to put static imports at the end of list, not at the beginning. See other class code for example.

Reply:
Hm, strange. I imported [hibernate-java-formatting.xml](https://raw.githubusercontent.com/hibernate/hibernate-ide-codestyles/master/eclipse/hibernate-java-formatting.xml) into Eclipse but this doesn't seem to do the trick. However, I'll apply your suggestion. Thank you!

Reply:
Ah, I see, [Bug 304415 - [organize imports] Organization of static imports in Java is not configurable](https://bugs.eclipse.org/bugs/show_bug.cgi?id=304415).

Reply:
thanks. other than that, your PR seems pretty nice.

Reply:
Thanks! What about the issue in `resultSetMaxValue()`? Should this be a different PR/issue?

Reply:
TBH, I am not sure. Sooner or later some team member (I am not) will review for this ticket is labelled as next release version. He will answer your question, I guess.

Reply:
Hi guys. Any chances we can get this PR merged soon?